apple/t2/default.nix

```diff
@@ -72,7 +72,7 @@ in
       # For audio
       boot.kernelParams = [ "pcie_ports=compat" "intel_iommu=on" "iommu=pt" ];
 
-      hardware.pulseaudio.package = overrideAudioFiles pkgs.pulseaudio "src/modules/";
+      services.pulseaudio.package = overrideAudioFiles pkgs.pulseaudio "src/modules/";
```

Comment:
This repository supports both nixos stable and unstable so this must be set conditional based on the nixpkgs version. Check out for instances of `lib.versionOlder` to learn how to include nixos module based on the nixpkgs version.